src/main/java/frc/robot/commands/shooter/AutoScore.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.commands.shooter;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ARM.AMP_STATE;
+import frc.robot.constants.INTAKE.INTAKE_STATE;
+import frc.robot.constants.SHOOTER.RPM_SETPOINT;
+import frc.robot.subsystems.AmpShooter;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Shooter;
+
+public class AutoScore extends Command {
+  private final Shooter m_shooter;
+  private final AmpShooter m_ampShooter;
+  private final Intake m_intake;
+  private final double m_percentOutput;
+  private final double allowableError = 500; // in RPM
+
+  public AutoScore(Shooter shooter, AmpShooter ampShooter, Intake intake, double percentOutput) {
+    m_shooter = shooter;
+    m_ampShooter = ampShooter;
+    m_intake = intake;
+    m_percentOutput = percentOutput;
+
+    addRequirements(m_shooter);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  @Override
+  public void execute() {
+    m_shooter.setRPMOutput(RPM_SETPOINT.SPEAKER.get());
+
+    if (m_shooter.getRpmMaster() >= RPM_SETPOINT.SPEAKER.get() - allowableError
```

Comment:
Are we sure that this if statement is going to check more than once?

Reply:
yes, it is ran in execute so it should be ran as long as the whole command is ran